target/linux/realtek/rtl839x/config-5.10

```diff
@@ -7,8 +7,8 @@ CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=16
 CONFIG_BLK_DEV_RAM_SIZE=4096
 # CONFIG_BMIPS_CPUFREQ is not set
-CONFIG_CEVT_R4K=y
-CONFIG_CEVT_RTL9300=y
+# CONFIG_CEVT_R4K is not set
+# CONFIG_CEVT_RTL9300 is not set
```

Comment:
Weird that this is in this commit as well; as I just left a comment on the previous commit :p